e2e/BKTClient.spec.ts

```diff
@@ -60,4 +64,90 @@ suite('e2e/BKTClientTest', () => {
       reason: 'RULE',
     })
   })
+
+  suite('forceUpdate', () => {
+    test('evaluatedAt is too old', async () => {
+      const client = getBKTClient()
+
+      assert(client != null)
+
+      const evaluationStorage = getDefaultComponent(
+        client,
+      ).dataModule.evaluationStorage() as EvaluationStorageImpl
+
+      const current = evaluationStorage.storage.get()
+
+      assert(current != null)
+
+      // update evaluations manually, and check if evaluation1 is deleted after fetchEvaluations
+      evaluationStorage.storage.set({
+        ...current,
+        evaluations: { [evaluation1.featureId]: evaluation1 },
+        evaluatedAt: '1',
+      })
+
+      const testTarget = evaluationStorage.storage.get()
+
+      assert(testTarget != null)
+      console.log(testTarget)
+
+      expect(testTarget.evaluatedAt).toBe('1')
+      expect(testTarget.evaluations[evaluation1.featureId]).toStrictEqual(
+        evaluation1,
+      )
+
+      await client.fetchEvaluations()
+
+      const updated = evaluationStorage.storage.get()
+
+      assert(updated != null)
+
+      expect(updated.evaluatedAt).not.toBe('1')
+      expect(updated.evaluations[evaluation1.featureId]).toBeUndefined()
+    })
+
+    test('userEvaluationId is null', async () => {
+      const client = getBKTClient()
+
+      assert(client != null)
+
+      const evaluationStorage = getDefaultComponent(
+        client,
+      ).dataModule.evaluationStorage() as EvaluationStorageImpl
+
+      const current = evaluationStorage.storage.get()
+
+      assert(current != null)
+
+      // update evaluations manually, and check if evaluation1 is deleted after fetchEvaluations
+      evaluationStorage.storage.set({
+        ...current,
+        evaluations: { [evaluation1.featureId]: evaluation1 },
+        currentEvaluationsId: null,
```

Comment:
On the other hand, request with empty evaluationsId  works as expected.

request:

```json
{
  "user":{"id":"bucketeer-js-user-id-1","attributes":{}},
  "userEvaluationsId":"",
  "userEvaluationCondition":{"evaluatedAt":"1689779952","userAttributesUpdated":false},
  "tag":"javascript",
  "sourceId":7,
  "sdkVersion":"2.0.2"
}
```

response:

```json
{
    "state": "FULL",
    "evaluations": {
        "id": "11556537758303297585",
        "evaluations": [
            {
                "id": "feature-javascript-e2e-boolean:2:bucketeer-js-user-id-1",
                "featureId": "feature-javascript-e2e-boolean",
                "featureVersion": 2,
                "userId": "bucketeer-js-user-id-1",
                "variationId": "3fd13e1e-a639-4a36-bccb-69a23445ea48",
                "variation": {
                    "id": "3fd13e1e-a639-4a36-bccb-69a23445ea48",
                    "value": "false",
                    "name": "",
                    "description": ""
                },
                "reason": {
                    "type": "OFF_VARIATION",
                    "ruleId": ""
                },
                "variationValue": "false",
                "variationName": ""
            },
            {
                "id": "feature-js-e2e-boolean:3:bucketeer-js-user-id-1",
                "featureId": "feature-js-e2e-boolean",
                "featureVersion": 3,
                "userId": "bucketeer-js-user-id-1",
                "variationId": "4fab39c8-bf62-4a78-8a10-1b8bc3dd3806",
                "variation": {
                    "id": "4fab39c8-bf62-4a78-8a10-1b8bc3dd3806",
                    "value": "true",
                    "name": "",
                    "description": ""
                },
                "reason": {
                    "type": "DEFAULT",
                    "ruleId": ""
                },
                "variationValue": "true",
                "variationName": ""
            },
            {
                "id": "feature-js-e2e-double:3:bucketeer-js-user-id-1",
                "featureId": "feature-js-e2e-double",
                "featureVersion": 3,
                "userId": "bucketeer-js-user-id-1",
                "variationId": "2d4a213c-1721-434b-8484-1b72826ece98",
                "variation": {
                    "id": "2d4a213c-1721-434b-8484-1b72826ece98",
                    "value": "2.1",
                    "name": "",
                    "description": ""
                },
                "reason": {
                    "type": "DEFAULT",
                    "ruleId": ""
                },
                "variationValue": "2.1",
                "variationName": ""
            },
            {
                "id": "feature-js-e2e-int:3:bucketeer-js-user-id-1",
                "featureId": "feature-js-e2e-int",
                "featureVersion": 3,
                "userId": "bucketeer-js-user-id-1",
                "variationId": "6079c503-c281-4561-b870-c2c59a75e6a6",
                "variation": {
                    "id": "6079c503-c281-4561-b870-c2c59a75e6a6",
                    "value": "10",
                    "name": "",
                    "description": ""
                },
                "reason": {
                    "type": "DEFAULT",
                    "ruleId": ""
                },
                "variationValue": "10",
                "variationName": ""
            },
            {
                "id": "feature-js-e2e-json:3:bucketeer-js-user-id-1",
                "featureId": "feature-js-e2e-json",
                "featureVersion": 3,
                "userId": "bucketeer-js-user-id-1",
                "variationId": "8b53a27b-2658-4f8c-925e-fb277808ed30",
                "variation": {
                    "id": "8b53a27b-2658-4f8c-925e-fb277808ed30",
                    "value": "{ \"key\": \"value-1\" }",
                    "name": "",
                    "description": ""
                },
                "reason": {
                    "type": "DEFAULT",
                    "ruleId": ""
                },
                "variationValue": "{ \"key\": \"value-1\" }",
                "variationName": ""
            },
            {
                "id": "feature-js-e2e-string:5:bucketeer-js-user-id-1",
                "featureId": "feature-js-e2e-string",
                "featureVersion": 5,
                "userId": "bucketeer-js-user-id-1",
                "variationId": "87e0a1ef-a0cb-49da-8460-289948f117ba",
                "variation": {
                    "id": "87e0a1ef-a0cb-49da-8460-289948f117ba",
                    "value": "value-1",
                    "name": "",
                    "description": ""
                },
                "reason": {
                    "type": "DEFAULT",
                    "ruleId": ""
                },
                "variationValue": "value-1",
                "variationName": ""
            }
        ],
        "createdAt": "1689779952",
        "archivedFeatureIds": [],
        "forceUpdate": true
    },
    "userEvaluationsId": "3227641913513702639"
}
```